github/github_test.go

```
@@ -0,0 +1,60 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestExecIssueList(t *testing.T) {
+	repo := "t-eckert/shepherd"
+
+	response, err := execIssueList(repo)
+
+	if err != nil {
+		fmt.Println(response)
+		fmt.Println(err)
+		t.Fail()
+	}
+}
+
+func TestParseIssuesToList(t *testing.T) {
+	issues := []byte("[{\"number\":2,\"title\":\"Issue 2\"},{\"number\":1,\"title\":\"Issue 1\"}]")
+
+	expected := []Issue{
+		{Number: 2, Title: "Issue 2"},
+		{Number: 1, Title: "Issue 1"},
+	}
+
+	actual, _ := parseIssuesToList(issues)
```

shouldn't swallow errors in tests. Instead check if and fail if it's there.